Source/RxSwift/sorted.swift

```diff
@@ -28,8 +36,9 @@ extension Observable where E: Sequence, E.Iterator.Element: Comparable {
      - returns: The sorted observable.
      */
     
-    public func sorted<T>(_ areInIncreasingOrder: @escaping (T, T) -> Bool) -> Observable<[T]> 
-        where E.Iterator.Element == T {
-        return self.map { $0.sorted(by: areInIncreasingOrder) }
+    public func sorted<T>(
+        _ using: @escaping (T,T) -> Bool
```

Comment:
The parameter should be `by areInIncreasingOrder: @escaping (T, T) -> Bool`, much like your original documentation and also like the original sorted method.